apps/web/src/pages/settings/organization/OrganizationName.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Input, successMessage } from '@novu/design-system';
+import { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useRenameOrganization } from '../../../api/hooks';
+import { css } from '../../../styled-system/css';
+import { Stack } from '../../../styled-system/jsx';
+
+type FormValues = {
+  name: string;
+};
+
+const INPUT_HEIGHT = '50px';
+
+export function OrganizationName({ organizationName }: { organizationName?: string }) {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking), the nittiest of nits:** Unnecessarily verbose given the name of the component‏
```suggestion
export function OrganizationName({ name }: { name?: string }) {
```